_includes/code/howto/java/src/test/java/io/weaviate/docs/model_providers/ConnectWeaviateEmbeddings.java

```diff
@@ -1,35 +1,32 @@
 package io.weaviate.docs.model_providers;
 
 // START-ANY
-
 import io.weaviate.client.Config;
 import io.weaviate.client.WeaviateAuthClient;
 import io.weaviate.client.WeaviateClient;
 import io.weaviate.client.base.Result;
 
+// END-ANY
 import static org.assertj.core.api.Assertions.assertThat;
 
 // START-ANY
-// Set these environment variables
-// WCD_HOSTNAME         Your Weaviate instance hostname
-// WCD_API_KEY          Your Weaviate instance API key
-
 public class ConnectWeaviateEmbeddings {
   public static void main(String[] args) throws Exception {
 
-    String host = System.getenv("WCD_W_EMB_HOSTNAME");
-    String apiKey = System.getenv("WCD_W_EMB_API_KEY");
+    // Best practice: store your credentials in environment variables
+    String wcdUrl = System.getenv("WEAVIATE_URL");      // Weaviate URL: "REST Endpoint" in Weaviate Cloud console
+    String wcdKey = System.getenv("WEAVIATE_API_KEY");  // Weaviate API key: "ADMIN" API key in Weaviate Cloud console
 
     // highlight-start
     Map<String, String> headers = new HashMap<String, String>() { {
-      put("X-Weaviate-Api-Key", apiKey);
-      put("X-Weaviate-Cluster-Url", "https://" + System.getenv("WCD_W_EMB_HOSTNAME"));
+      put("X-Weaviate-Api-Key", wcdKey);
+      put("X-Weaviate-Cluster-Url", "https://" + wcdUrl);
```

Comment:
Here using `hostname` would also help to distinguish between the string with the scheme & without - so people don't end up with the full URL prepended with the scheme. :)